editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5222,6 +5222,9 @@ CanvasItemEditor::CanvasItemEditor() {
 	snap_target[0] = SNAP_TARGET_NONE;
 	snap_target[1] = SNAP_TARGET_NONE;
 
+	horizontal_guides_meta = StaticCString::create("_edit_horizontal_guides_");
+	vertical_guides_meta = StaticCString::create("_edit_vertical_guides_");
```

Comment:
Out of curiosity, could the initialization be moved to the header? I thought this would be possible with `SNAME()`.